package com.cohen.wechat.app.server.common.test.exception;

/**
 * @Author: linjincheng
 * @createTime: Created in 2018/11/30 14:16
 * @version: v1.0.0
 * @copyright: choice
 * @email: ljc@choicesoft.com.cn
 */
public class Test2 {
    public void test() {
        throw new RuntimeException("hello");
    }
}